packages/react-magma-dom/src/components/Pagination/SimplePagination.test.js

```diff
@@ -0,0 +1,178 @@
+
+    const selected = getByText('2').parentElement.parentElement;
+
+    expect(selected).toHaveStyleRule(
+      'background-color',
+      magma.colors.neutral100
+    );
+    expect(selected).toHaveStyleRule('border-radius', '8px');
+  });
+
+  it('Should render an inverse select', () => {
+    const { getByText } = render(<Pagination count={4} simple isInverse />);
+
+    const selected = getByText('2').parentElement.parentElement;
+
+    expect(selected).toHaveStyleRule(
+      'background-color',
+      transparentize(0.8, magma.colors.neutral900)
+    );
+  });
+
+  it('Should render disabled navigation icons, text, and select', () => {
+    const { getByText, getByLabelText } = render(
+      <Pagination simple count={4} disabled />
+    );
+    const previousButton = getByLabelText('Previous Page');
+    const nextButton = getByLabelText('Next Page');
+
+    const selected = getByText('2').parentElement;
+
+    expect(selected).toHaveStyleRule('cursor', 'not-allowed');
+
+    expect(selected).toHaveStyleRule(
+      'color',
+      transparentize(0.4, magma.colors.neutral500)
+    );
+
+    expect(previousButton).toHaveStyleRule(
+      'color',
+      transparentize(0.4, magma.colors.neutral500)
+    );
+    expect(nextButton).toHaveStyleRule(
+      'color',
+      transparentize(0.4, magma.colors.neutral500)
+    );
+  });
+
+  it('Should render inverse disabled navigation icons, text, and select', () => {
+    const { getByText, getByLabelText } = render(
+      <Pagination simple count={4} isInverse disabled />
+    );
+    const previousButton = getByLabelText('Previous Page');
+    const nextButton = getByLabelText('Next Page');
+
+    const selected = getByText('2').parentElement;
+
+    expect(selected).toHaveStyleRule('cursor', 'not-allowed');
+
+    expect(selected).toHaveStyleRule(
+      'color',
+      transparentize(0.6, magma.colors.neutral100)
+    );
+
+    expect(previousButton).toHaveStyleRule(
+      'color',
+      transparentize(0.7, magma.colors.neutral100)
+    );
+    expect(nextButton).toHaveStyleRule(
+      'color',
+      transparentize(0.7, magma.colors.neutral100)
+    );
+  });
+
+  it('Should disable the previous button upon selecting the last page', () => {
+    const { getByLabelText } = render(
+      <Pagination simple count={4} defaultPage={2} testId={testId} />
+    );
+    let previousButton = getByLabelText('Previous Page');
+
+    fireEvent.click(previousButton);
+
+    expect(previousButton).toHaveStyleRule(
+      'color',
+      transparentize(0.4, magma.colors.neutral500)
+    );
+  });
+
+  it('Should disable the next button upon selecting the last page', () => {
+    const { getByLabelText } = render(
+      <Pagination simple count={4} defaultPage={3} testId={testId} />
+    );
+    let nextButton = getByLabelText('Next Page');
+
+    fireEvent.click(nextButton);
+
+    expect(nextButton).toHaveStyleRule(
+      'color',
+      transparentize(0.4, magma.colors.neutral500)
+    );
```

Comment:
In addition to checking the color changes, can we check that the element is disabled?